pkg/scheduler/server/internal/pool.go

```diff
@@ -26,7 +26,10 @@ import (
 	"github.com/dapr/kit/logger"
 )
 
-var log = logger.NewLogger("dapr.runtime.scheduler")
+var (
+	issueSendingJobToDapr = atomic.Bool{}
```

Comment:
Why global?

Reply:
fair, let me change that

Reply:
oh wait, we want it for the entire process lifetime to only log once

Reply:
I still don't think this should be a global variable.